notebooks/tutorial/7 - Getting data from the ADX.ipynb

```diff
@@ -0,0 +1,705 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "edeba4c2",
+   "metadata": {},
+   "source": [
+    "# Data Volume\n",
+    "\n",
+    "The volume of data and number of files is quite large for this dataset. For example, here are the number of matches in each month for the first five months:\n",
+    "\n",
+    "- December 2021: 17,952\n",
+    "- January 2022: 15,315\n",
+    "- February 2022: 9,892\n",
+    "- March 2022: 16,855\n",
+    "- April 2022: 8,717\n",
+    "\n",
+    "\n",
+    "We can use the month of April to demonstrate the volume. The 30 data revisions in april in total had:\n",
+    "\n",
+    "- 286,671 files\n",
+    "- 8,717 matches\n",
+    "- 275 GB\n",
+    "\n",
+    "These values are used in the calculator below to estimate costs on a per-revision basis. Though, the number of matches per day is variable.  For the first batch of historical revisions, you can find [the approximate number of matches for each day](https://docs.pureskill.gg/datascience/adx/csgo/csds/revisions). There is quite a large variance in some days due to spikes in user activity.\n",
+    "\n",
+    "------------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29b4d737",
+   "metadata": {},
+   "source": [
+    "# 1. Subscribe to the Dataset on the ADX\n",
+    "\n",
+    "The first step to getting the data is to subscribe to the [Product](https://aws.amazon.com/marketplace/pp/prodview-v3o7zrt6okwmo) on the ADX. \n",
+    "\n",
+    "Once your subscription has been approved, you can get your dataset ID by going to the ADX, clicking on \"Entitled data\", going to \"PureSkill.gg Competitive CS:GO Gameplay\", clicking on \"pureskillgg-csgo-production-dataexchange-csds-0\" under \"Entitled data sets\", expanding the \"Data set overview\" box, and copying the \"Data set ID\" on the right. Paste that into the `dataset_id` variable below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "52a341d4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset_id = \"f49be2ef387af522a7b6f000158113e0\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c5161b02",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if dataset_id == \"f49be2ef387af522a7b6f000158113e0\":\n",
+    "    raise Exception(\"Your dataset_id is still the default. Please subscribe on the ADX\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6603c1ae",
+   "metadata": {},
+   "source": [
+    "# 2. Estimate Costs \n",
+    "\n",
+    "We provide the estimates below for convienence but do not guarantee their accuracy or applicability to your AWS account. You should perform your own calculations using the cost calculator provided by AWS."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab98b929",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datetime import date\n",
```

Comment:
```suggestion
    "from datetime import date\n",
    "\n",
```

Reply:
Moved to cell below to keep the one the user should edit clean.